<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnnotationToolMainWindow</class>
 <widget class="QMainWindow" name="AnnotationToolMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PG Speech Transcriber</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="PticaGovorun::FileWorkspaceWidget" name="widgetFileWorkspace" native="true"/>
      <widget class="QSplitter" name="splitterContentUpBottom">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="widgetUpperPanel" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <widget class="QLineEdit" name="lineEditRecognizerName">
         <property name="geometry">
          <rect>
           <x>109</x>
           <y>2</y>
           <width>71</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>recog name</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonSaveAudioAnnot">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>41</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonSegmentComposerPlay">
         <property name="geometry">
          <rect>
           <x>54</x>
           <y>0</y>
           <width>51</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>SegPlay</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
       <widget class="QTabWidget" name="tabWidgetSpeechTranscriptionTabs">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <property name="tabsClosable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
        </widget>
       </widget>
       <widget class="QWidget" name="widgetBottomPanel" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QSplitter" name="splitterBottomLeftRight">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QPlainTextEdit" name="plainTextEditLogger">
            <property name="undoRedoEnabled">
             <bool>false</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QPlainTextEdit" name="plainTextEditAudioSegmentsComposer"/>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>830</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenAnnotDir"/>
    <addaction name="actionCloseAnnotDir"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpenAnnotDir">
   <property name="text">
    <string>Open Annot Dir</string>
   </property>
   <property name="shortcut">
    <string extracomment="Open directory with speech annotations">F3</string>
   </property>
  </action>
  <action name="actionCloseAnnotDir">
   <property name="text">
    <string>Close Annot Dir</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PticaGovorun::FileWorkspaceWidget</class>
   <extends>QWidget</extends>
   <header>FileWorkspaceWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
